src/AsyncMqttClient.cpp

```diff
@@ -310,7 +310,11 @@ void AsyncMqttClient::_onConnect(AsyncClient* client) {
     return;
   }
 
-  _client.add(fixedHeader, 1 + remainingLengthLength);
+  char* buff = new char[neededSpace];
+  size_t pos = 0;
+
+  memcpy(&buff[pos], fixedHeader, 1 + remainingLengthLength);
+  pos += 1 + remainingLengthLength;
 
   // Using a sendbuffer to fix bug setwill on SSL not working
   char sendbuffer[12];
```

Comment:
I would really like to see something like https://github.com/mknjc/async-mqtt-client/blob/04df4e00f55b0ffd7a6872c48c386428bdff924f/src/AsyncMqttClient/Headers.hpp#L7 to make it more readable

Reply:
> I would really like to see something like https://github.com/mknjc/async-mqtt-client/blob/04df4e00f55b0ffd7a6872c48c386428bdff924f/src/AsyncMqttClient/Headers.hpp#L7 to make it more readable

Off course. I just wanted a quick bugfix.